content/cloud/pricing-and-billing.md

```diff
@@ -109,6 +109,6 @@ Fermyon will notify you via email when you reach 75%, 90%, and 100% usage for yo
 
 Please reach out to [our team](mailto:sales@fermyon.com) and we'll create a customized solution to meet your current needs. 
 
-**I'm interested in increase a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**
+**I'm interested in increasing a quota limit on a resource not listed in Fermyon's pricing plan, can you help?**
```

Comment:
Is this any different than the above question? I can see the nuance on the technical implementation, but just having one Q about raising quota limits, seems to be sufficient.

Reply:
I reworked the first question to make the distinction more clear. The first question is around promoting from Growth Plan to Enterprise Plan. The second question is about increasing quota limits not explicitly included in the pricing model - this could mean the user is a good fit for Enterprise, but it could also mean they have requirements we aren't able to meet quite yet as a Cloud (due to them requesting an increase in a resource we can't scale et), or that our pricing model should include that resource going forward.

Does that help separate them a bit more, or should we consider consolidating the question?